google/cloud/internal/timer_queue.cc

```diff
@@ -32,9 +41,10 @@ future<StatusOr<std::chrono::system_clock::time_point>> TimerQueue::Schedule(
   bool should_notify = false;
   {
     std::lock_guard<std::mutex> lk(mu_);
-    if (shutdown_)
-      return make_ready_future(StatusOr<std::chrono::system_clock::time_point>(
-          Status(StatusCode::kCancelled, "TimerQueue already shutdown")));
+    if (shutdown_) {
+      return make_ready_future(
+          MakeCancelled("TimerQueue already shutdown", GCP_ERROR_INFO()));
```

Comment:
Q. Is it worth distinguishing (in the status message) between a pre- and post-Shutdown() cancellation?  That is, s/ already// perhaps?  If `Schedule()` calls aren't already synchronized with `Shutdown()`, then it is just a race anyway.

Reply:
Done.